scripts/model_player.py

```diff
@@ -0,0 +1,70 @@
+from random import random
+
+from scripts.game_session import GameSession
+from scripts.models import Estimator, probability_A, Qlearning, RescorlaWagner
+
+
+class ModelPlayer:
```

Comment:
as here we have huge duplication I think we should inherit from VirtualPlayer and rewrite play method of VirtualPlayer in a way that it could be easily reused here